ciongke/cmd/hook/githubagent.go

```diff
@@ -52,9 +52,9 @@ type JenkinsJob struct {
 	// eg @k8s-bot e2e test this
 	Trigger *regexp.Regexp
 	// Whether or not to comment on GitHub on failure.
-	CommentOnFailure bool
+	SkipFailureComment bool
```

Comment:
Done, although keeping the flag for now and also sending the flag until the next push per standard flag deprecation practice.